### DISCUSSION TITLE
Idea: bulk beatmap selection

### DISCUSSION THREAD

**Discussion by Gamingzone123** [ppy/osu#17117] [Ideas]
Idea/Suggestion: add a feature to bulk select beatmaps for adding to collection/deleting/etc, bulk collection/beatmap export for people moving pc's etc. maybe through a new file type like osz for groups of beatmaps, new file type could also potentially be utilized further with beatmap packs?

side note: along a similar vein, a beatmap sync feature for those on multiple devices would also be nice, especially when osu is pushed to mobile and people want their maps on both devices without having to individually download the same map on both

**Comment by TheFantasticLoki** [comment]
I swear I have seen both of these ideas in previous discussions but can't find anything in my search someone else mind tagging them? Ik they exist just can't seem to find them.

**Reply by Gamingzone123** [reply]
i also did a search but couldn't find them
